server/repository/src/mock/test_requisition_service.rs

```diff
@@ -198,6 +201,34 @@ pub fn mock_draft_response_requisition_for_update_test_line() -> RequisitionLine
     })
 }
 
+pub fn mock_full_draft_response_requisition_for_update_test() -> FullMockRequisition {
```

Comment:
Kinda hacky since API doesn't allow `Response Requisitions` to have a `DRAFT` status... Created to pass `NotARequestRequisition` error... that should never happen anyways but have kept the error in to be consistent with invoice/response requisition errors...

Reply:
Was this already a problem/hacky before?

Reply:
Yeah it's testing for DRAFT status, I just moved it so it doesn't fail the translator tests